Deepanshu_Balani/project2 tictactoe/assets/tic-tac.js

```diff
@@ -0,0 +1,108 @@
+
+var N_SIZE = 3,
```

Comment:
is N_SIZE a constant? if yes why var ?

Reply:
i have changed declaration way and made N_SIZE const